src/InvariantTest.sol

```diff
@@ -35,6 +47,24 @@ contract InvariantTest is Test {
         excludedSenders_ = _excludedSenders;
     }
 
+    function targetArtifact(string memory newTargetedArtifact_) internal {
+        _targetedArtifacts.push(newTargetedArtifact_);
+    }
+
+    function targetArtifacts() public view returns (string[] memory targetedArtifacts_) {
+        require(_targetedArtifacts.length != uint256(0), "NO_TARGETED_ARTIFACTS");
```

Comment:
How come the artifact handlers have `require` statements to ensure non-zero length but others don't? Feels like they should be consistent. Maybe a @joshieDo question

Reply:
afaik there's no need for that